docs/style-guide.md

```diff
@@ -1,30 +1,30 @@
 # Code style guide
 
-The document introduces the style guide for codes submited to poi. Codes of good and coherent style is friendly to other contributors and yourself in the future. Please follow them.
+This document introduces the style guide for code submitted to poi. Good and coherent code style is friendly to other contributors and yourself in the future. Please follow them.
```

Comment:
To be honest, even after the changes I still feel awkward after I read this part out. This definitely has a lot of room for improvement.